observers/observer-config.default.json

```diff
@@ -0,0 +1,54 @@
+{
+  "globals": {
+    "goose_provider": "ollama",
+    "goose_model": "gpt-oss:20b",
```

Comment:
@AaronGoldsmith if not set - will it use system default? as maybe that is what we want as default (ie don't assume ollama?)

Reply:
@michaelneale, I believe i've addressed this in 34a4ee4e54118b9958a0ced04232b3500fa09dfb, let me know if we should simplify the logic further